src/features/profile/berths/Berths.tsx

```diff
@@ -1,45 +1,148 @@
-import React from 'react';
+import { useTranslation } from 'react-i18next';
```

Comment:
Shouldn't React be imported here?

Reply:
With the new JSX transformation, importing React is [no longer needed](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform).

This is covered by version 16.4 which we're using.
> A version of React that supports the new transform (React 17 RC and higher supports it, but we’ve also released React 16.14.0, React 15.7.0, and React 0.14.10 for people who are still on the older major versions).

As for this PR, I will remove the import statements from the rest of the files.

Reply:
TIL, good job 👍